standards/domains.md

```diff
@@ -0,0 +1,49 @@
+---
+category: Operating services
+---
+# Domains
+
+All services with real users and which look like GOV.UK should be on
+`*.service.gov.uk` or `*.service.justice.gov.uk`.
+
+### service.gov.uk
+
+The [service standard](https://www.gov.uk/service-manual/technology/get-a-domain-name)
+already sets out when and how to get a `*.service.gov.uk` subdomain -
+public-facing services which have passed their beta assessment should
+be able to get one.
+
+### service.justice.gov.uk
+
+All other services with real users and which look like GOV.UK should be
+on here, including:
+
+- public-facing services which haven’t yet passed their beta assessment
+(but may be in private beta with a restricted set of real users)
+- internal services which look like GOV.UK (although [they can’t use the
+crown, New Transport font etc](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk))
+
+Any staff-facing sites which exist as part of public-facing services
```

Comment:
Just wondering whether you want to make this:
```suggestion
## Staff-facing sites

Any staff-facing sites which exist as part of public-facing services
```
In the same way you have pre-production environments as a separate heading below.  

The guidance applies to both of the cases above but currently sits under one of them -- people might not read it if they are only interested in the `service.gov.uk` bit.

Reply:
Thanks, I've done this.